packages/func/src/executors/common/get-copy-package-to-app-transformer-factory.ts

```diff
@@ -32,7 +32,11 @@ export const getCopyPackageToAppTransformerFactory = (context: ExecutorContext)
       if (isImportDeclaration(node)) {
         const cleanedModuleName = node.moduleSpecifier.getText().replace(/['"]/g, '');
         const moduleInOriginalPackageJson = findModule(originalPackageJson.dependencies, cleanedModuleName);
-        // If the original package.json has the dependency, copy it to the app package.json
+         // Create a dependencies section in the app package.json if it doesn't exist
+        if (!appPackageJson.dependencies) 
+          appPackageJson.dependencies = {};
```

Comment:
The `visitChild` will be called on each node in each file of the program.
I think it would be better to have the if statement happen outside of the `visitChild`, so it would run only once.

Reply:
I see what you mean, but for this fix I considered it an unnecessary optimisation because even if a falsey nullish check were executed for thousands of nodes it wouldn't have any discernible affect on performance.